ngraph_creator/include/NgraphNodes.hpp

```diff
@@ -36,6 +36,7 @@ class NgraphNodes {
     const std::string& getNodeName(size_t index);
 
     std::shared_ptr<ngraph::Function> generateGraph();
+    void setNullPtr(size_t index);
```

Comment:
Can you change this to setInvalidNode.
Also, leave a comment here that :
Setting the node name to empty string "". Caller of getNodeName should validate against "".